src/scss/components/switch/_index.scss

```diff
@@ -0,0 +1,185 @@
+@import "./config";
+
+hx-switch-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: auto 1fr;
+  -ms-grid-rows: auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "ctrl lbl"
+    ".    help";
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto;
+  justify-items: start;
+  grid-area: facade;
+  margin-top: 0.25rem; // FIXME: fix with proper inline styling
+  position: relative;
+  width: 68px;
+  height: 32px;
+
+  > input[type="checkbox"] {
```

Comment:
@lalithkarikelli, here is `<hx-switch>` with the label attributes:

```html
<hx-switch onlabel="yes" offlabel="no"></hx-switch>
```

To select the above, I had to increase CSS Specificity:

```css

/**** Switch Off ****/
hx-switch-control > input[type="checkbox"] ~ label > hx-switch[offlabel]::before {
  content: attr(offlabel);
  text-transform: uppercase;

  /* Switch Off styles and states here */
}

/**** Switch On ****/
hx-switch-control > input[type="checkbox"]:checked ~ label > hx-switch[onlabel]::before {
    content: attr(onlabel);
    text-transform: uppercase;

  /* Switch On styles and states here */

 }

```

Reply:
Changes have been done, please review.